docs/orchestrating-elastic-stack-applications/stack-config-policy.asciidoc

```diff
@@ -35,7 +35,7 @@ Elastic Stack configuration policies can be defined in a `StackConfigPolicy` res
 * `name` is a unique name used to identify the policy.
 * `spec.elasticsearch` describes the settings to configure and at least one setting must be defined. Each of the following fields except `clusterSettings` is an associative array where keys are arbitrary names and values are definitions:
   ** `clusterSettings` are the settings that go into the elasticsearch.yml file.
-  ** `snapshotRepositories` are snapshot repositories for defining an off-cluster storage location for your snapshots.
+  ** `snapshotRepositories` are snapshot repositories for defining an off-cluster storage location for your snapshots (check <<{p}-{page_id}-specifics-snap-repo,specifics>>).
```

Comment:
nit: I have a small preference for the link name to be automatically synchronized:

```suggestion
  ** `snapshotRepositories` are snapshot repositories for defining an off-cluster storage location for your snapshots. Check <<{p}-{page_id}-specifics-snap-repo>> for more information.
```

⬇️ 

<img width="773" alt="image" src="https://user-images.githubusercontent.com/976373/232986821-3e71ea89-66ef-457c-a2e3-547f659f55ea.png">